src/process_definition_repository.ts

```diff
@@ -27,8 +31,23 @@ export class ProcessDefinitionRepository implements IProcessDefinitionRepository
   }
 
   public async initialize(): Promise<void> {
+    logger.verbose('Initializing Sequelize connection and loading models...');
+    const connectionAlreadyEstablished: boolean = this._sequelize !== undefined;
+    if (connectionAlreadyEstablished) {
+      logger.verbose('Repository already initialized.');
```

Comment:
```suggestion
      logger.verbose('Repository already initialised. Done.');
```

Just for consistency.

Reply:
It's `initialized`.
https://www.dict.cc/?s=initialized